tools/ci/bench-jmh.sh

```diff
@@ -6,6 +6,7 @@ source "$(dirname "$0")/common.sh"
 
 java -jar "$RENAISSANCE_JMH_JAR" \
 	-jvmArgs -Xms2500M -jvmArgs -Xmx2500m \
+	$( for arg in $( get_jvm_workaround_args ); do echo "-jvmArgs" "$arg"; done ) \
```

Comment:
Unless we put other workaround args in here, I'd suggest to just name it `get_jvm_add_opens_args` for now.

Reply:
I would say that the fact that it adds only `--add-opens` is an implementation detail from the point of view of the wrapper script.